test/models/GetStateBenefitsModelSpec.scala

```diff
@@ -0,0 +1,440 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import com.codahale.metrics.SharedMetricRegistries
+import models.GetStateBenefitsModel._
+import play.api.libs.json.{JsValue, Json}
+import utils.TestUtils
+
+
+
```

Comment:
This is a bit pedantic but if you are changing stuff anyway there is extra line spacing here not required